contracts/contracts/core/BlockTracker.sol

```diff
@@ -72,13 +73,13 @@ contract BlockTracker is IBlockTracker, BlockTrackerStorage,
     /**
      * @dev Records a new L1 block and its winner.
      * @param _blockNumber The number of the new L1 block.
-     * @param _winnerGraffiti The graffiti of the winner of the new L1 block.
+     * @param _winnerBLSKey The BLS key of the winner of the new L1 block.
      */
     function recordL1Block(
         uint256 _blockNumber,
-        string calldata _winnerGraffiti
+        bytes calldata _winnerBLSKey
     ) external onlyOracle whenNotPaused {
-        address _winner = blockBuilderNameToAddress[_winnerGraffiti];
+        address _winner = _providerRegistry.getEoaFromBLSKey(_winnerBLSKey);
```

Comment:
I plan to remove the concept of a builder in the blocktracker, retaining only the BLS key responsible for building the block number. However, due to an ongoing audit, I will maintain the current approach to minimize changes to the contract architecture.